clientconn.go

```diff
@@ -675,23 +694,81 @@ func (cc *ClientConn) scWatcher() {
 	}
 }
 
+// addressesUpdated checks whether current address in the updating list, Update the list if true.
+func (cc *ClientConn) addressesUpdated(addrs []Address) bool {
```

Comment:
Name this as pickFirstUpdateAddresses() and comment that it's only used when balancer is pickfirst.